### DISCUSSION TITLE
<Windows 10 start menu theme-aware tiles on and off>

### DISCUSSION THREAD

**Discussion by XuanChen1234** [community/community#62131] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Windows 10 start menu transparent tile is too ugly, how to turn off theme-aware tiles with code?
![Annotation 2023-06-06 093239 (2)](https://github.com/community/community/assets/140480110/327551bc-c672-468f-a39a-5a12b29a78de)

**Comment by queenofcorgis** [comment]
Hi @XuanChen1234 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on Windows like their [community](https://techcommunity.microsoft.com/t5/windows/ct-p/Windows). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
